lib/Widget/appointment/editAppointment.dart

```diff
@@ -449,7 +449,8 @@ class _EditAppointmentPageState extends State<EditAppointmentPage> {
                                                             'Appointments',
                                                         docId: appointmentId,
                                                         data: dataToDB);
-                                                Navigator.pop(context);
+                                                Navigator.pushNamed(context,
+                                                    '/appointment_page');
```

Comment:
This means user can go back via backarrow icon in the web browser? But, `appointment` is acting  like a Modal. So, I think it should just pop the navigator route out.